minecraft.js

```diff
@@ -66,6 +66,25 @@ app.post('/command', (req, res) => {
   }, 250);
 });
 
+app.get('/properties', (req, res) => {
+  const text = fs.readFileSync('./server.properties').toString('utf-8');
```

Comment:
You already mentioned this, but yeah don't use the sync version.

Reply:
You can make a promise version of the async version so you can use async/await

Reply:
Alrighty, I've made this change!